modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,488 @@
+= Configure a Customer-Managed VPC on GCP
+:description: Connect Redpanda Cloud to your existing VPC resources for additional security.
+:page-cloud: true
+
+You can connect Redpanda Cloud to your customer-managed virtual private cloud (VPC), and you can then create clusters with the most security. With standard BYOC clusters using a Redpanda-managed VPC, Redpanda creates a VPC in your Google Cloud account and manages its resources. With a customer-managed VPC:
+
+* You provide your own VPC in your Google Cloud account.
+* You maintain more control of your Google Cloud account, because Redpanda requires less permissions than standard BYOC clusters.
+* You control your security resources and policies, including subnets, service accounts, IAM roles, firewall rules, and storage buckets.
+
+[NOTE]
+====
+include::shared:partial$feature-flag.adoc[]
+====
+
+== Prerequisites
+
+* Redpanda creates a private Google Kubernetes Engine (GKE) cluster in the VPC. For more information, see the https://cloud.google.com/kubernetes-engine/docs/how-to/service-accounts[GKE service account documentation^].
```

Comment:
Darn--I don't see a prereq here. Exactly what is it? Should this be: 

You must have a Google Kubernetes Engine (GKE) cluster because Redpanda creates a private Google Kubernetes Engine (GKE) cluster in the VPC.

Reply:
reworded this! You're correct that there's no prereq, just a link to Google for info about GKE service account (so we don't need to explain it ourselves)